playbook/app/pb_kits/playbook/pb_dialog/_dialog.scss

```diff
@@ -68,7 +68,7 @@
 	$opacity_visible: 1;
 	$opacity_hidden: 0;
 
-	@include pb_card;
+	// @include pb_card;
```

Comment:
Was this causing an issue on the rails side? 
Curious how the react side looks without this.

Reply:
This was conflicting with the dialog's own styling, essentially  preventing it from being hidden when not active. I commented it out instead of deleting it for now just in case, but I did test all the react variants and they all seem to be looking fine without this inclusion. You can test it yourself in the milano env!